pkg/webhooks/pod_mutating_webhook.go

```diff
@@ -50,7 +50,7 @@ func (p *podWebhook) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-// InjectDecoder, when defined, will result in the decoder automatically being set.
+// InjectDecoder when defined, will result in the decoder automatically being set.
```

Comment:
Can you add this comma back actually please

Reply:
Thank you for your comment, I will modify it again

Reply:
There should be no similar errors like this in the project.